pkg/cmd/server/server.go

```diff
@@ -606,6 +606,8 @@ func (s *server) runControllers(config *api.Config, defaultBackupLocation *api.B
 			backupTracker,
 			s.sharedInformerFactory.Ark().V1().BackupStorageLocations(),
 			s.config.defaultBackupLocation,
+			nil,
```

Comment:
you can pass pass the VSL informer here (`s.sharedInformerFactory.Ark().V1().VolumeSnapshotLocations()`), just leave the line below as `nil` 'til wayne's PR goes in